test/networks/docker_compose/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3.2'
+
+services:
+  node0:
+    image: cheqd-node
+    ports:
+      - "26656:26656" # p2p
+      - "26657:26657" # rpc
+    volumes:
+      - ./node_configs/node0:/cheqd:Z
```

Comment:
The same question. It mounts directory `node0` into `/cheqd` but does node0 contains '.cheqdnode' inside?

Reply:
Yep, I changed Config generation script as well.